src/Compositor.cpp

```diff
@@ -2795,6 +2795,7 @@ void CCompositor::moveWindowToWorkspaceSafe(PHLWINDOW pWindow, PHLWORKSPACE pWor
     pWindow->updateToplevel();
     pWindow->updateDynamicRules();
     pWindow->uncacheWindowDecos();
+    pWindow->updateGroupOutputs();
 
     if (!pWindow->m_sGroupData.pNextWindow.expired()) {
```

Comment:
I wonder if this block can be removed, as:
+ `next->moveToWorkspace(pWorkspace);` is done in `pWindow->updateGroupOutputs();`
+ `next->updateToplevel();` is not done in `pWindow->updateGroupOutputs();.` Why is this needed here? could it be moved to `pWindow->updateGroupOutputs();` or is it a specific necessity for this function?